.github/workflows/release.yml

```diff
@@ -41,12 +42,35 @@ jobs:
         # TODO: add back 'windows-latest' when it can be compiled.
         os: ['macos-latest', 'ubuntu-18.04']
         py-version: ['3.6', '3.7', '3.8']
-        tf-version: ['2.4.1']
+        tf-version: ['1.15.2', '2.4.1']
         tf-need-cuda: ['1', '0']
+        tf-cuda-version: ['10.0', '11.0']
+        tf-cudnn-version: ['7', '8']
```

Comment:
7->7.4，is it better?
TensorFlow official website recommends：https://tensorflow.google.cn/install/source_windows?hl=en#gpu